<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1166</width>
    <height>706</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>GrifoT - Transferencias</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Fondo de la ventana */
QMainWindow {
    background-color: #F4F4F4;
}

/* ==========================
   MENU BAR (barra superior)
   ========================== */
QMenuBar {
    background-color: #33373B;
    color: #FFFFFF;
    font-size: 14px;
    font-weight: bold;
}

QMenuBar::item {
    background: transparent;
    padding: 12px 24px;
}

QMenuBar::item:selected { /* Hover */
    background: #2ABF9E;
    color: #FFFFFF;
    border-radius: 5px;
}

QMenuBar::item:pressed {
    background: #1F8C72;
}

/* ==========================
   MENÚS DESPLEGABLES
   ========================== */
QMenu {
    background-color: #33373B;
    color: #FFFFFF;
    border: 1px solid #2ABF9E;
    padding: 5px;
}

QMenu::item {
    padding: 6px 20px;
    background: transparent;
}

QMenu::item:selected { /* Hover */
    background: #2ABF9E;
    border-radius: 5px;
}

/* ==========================
   TOOLBAR (acciones con icono/botón)
   ========================== */
QToolBar {
    background: #33373B;
    border: none;
    spacing: 6px;
    padding: 4px;
}

/* Los QAction en la toolbar aparecen como QToolButton */
QToolButton {
    background: transparent;
    color: #FFFFFF;
    padding: 8px 12px;
    border-radius: 8px;
    font-size: 14px;
    font-weight: bold;
}

QToolButton:hover {
    background: #2ABF9E;
    color: #FFFFFF;
}

QToolButton:pressed {
    background: #1F8C72;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="minimumSize">
       <size>
        <width>184</width>
        <height>516</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(51, 55, 59);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Información</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>91</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 3px solid #2ABF9E;
border-radius: 3px;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources/imagenes.qrc">:/xy/images/user.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Usuario:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Turno activo :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>250</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Módulo</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>300</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Ventana</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblEmpleado">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>150</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>User</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblTurnoActivo">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>170</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>turno</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(244, 244, 244);
color: rgb(29, 29, 29);</string>
       </property>
       <property name="text">
        <string>Dashboard General</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>330</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(244, 244, 244);
color: rgb(29, 29, 29);</string>
       </property>
       <property name="text">
        <string>Principal</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(244, 244, 244);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>100</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pagePrincipal">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="6" column="0">
            <widget class="QLabel" name="label_9">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>200</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../resources/imagenes.qrc">:/xy/images/banner_tusino.jpg</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="0">
            <widget class="QTableWidget" name="tblTanques">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>200</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>950</width>
               <height>300</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>-1</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QTableWidget {
    background-color: #1e2a38; /* azul oscuro */
    color: #e0f7e9; /* texto verde muy claro */
    border: 2px solid #2ecc71; /* verde */
    gridline-color: #2ecc71;
    font-size: 14px;
    border-radius: 8px;
}

QHeaderView::section {
    background-color: #2ecc71; /* verde */
    color: #1e2a38; /* azul oscuro para contraste */
    padding: 6px;
    font-weight: bold;
    border: none;
}

QTableWidget::item {
    selection-background-color: #2ecc71; 
    selection-color: #1e2a38;
}

QScrollBar:vertical {
    background: #1e2a38;
    width: 10px;
    margin: 0px;
    border-radius: 5px;
}

QScrollBar::handle:vertical {
    background: #2ecc71;
    min-height: 20px;
    border-radius: 5px;
}
</string>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
             </property>
             <property name="updateThreshold">
              <number>200</number>
             </property>
             <property name="rowCount">
              <number>5</number>
             </property>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>200</number>
             </attribute>
             <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderCascadingSectionResizes">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderHighlightSections">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <column>
              <property name="text">
               <string>id_inventario</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Nº Tanque</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Litros disponibles</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="7" column="0">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>12</pointsize>
               <italic>false</italic>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>PROGRAMA INTEGRAL - ESTACIÓN DE SERVICIO EL TUSINO</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_12">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Estado de los tanques</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Policy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>50</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page"/>
         <widget class="QWidget" name="page_2"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>184</width>
        <height>516</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(51, 55, 59);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>151</width>
         <height>151</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources/imagenes.qrc">:/xy/images/logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1166</width>
     <height>44</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="btnRegistrar_Transferencias"/>
    <addaction name="btnReportar_Transferencia"/>
    <addaction name="btnHistorial_de_Transferencias"/>
   </widget>
   <widget class="QMenu" name="menuVentas">
    <property name="title">
     <string>Ventas</string>
    </property>
    <addaction name="btnVentas"/>
   </widget>
   <widget class="QMenu" name="menuTurnos">
    <property name="title">
     <string>Turnos</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAdministraci_n">
    <property name="title">
     <string>Administración</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuReportes">
    <property name="title">
     <string>Reportes</string>
    </property>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuVentas"/>
   <addaction name="menuTurnos"/>
   <addaction name="menuAdministraci_n"/>
   <addaction name="menuReportes"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDesarrolladores">
   <property name="text">
    <string>Actualizar inventario</string>
   </property>
  </action>
  <action name="btnRegistrar_Transferencias">
   <property name="text">
    <string>Registrar Transferencias</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="btnReportar_Transferencia">
   <property name="text">
    <string>Reportar Transferencia</string>
   </property>
  </action>
  <action name="btnHistorial_de_Transferencias">
   <property name="text">
    <string>Historial de Transferencias</string>
   </property>
  </action>
  <action name="btnVentas">
   <property name="text">
    <string>Punto de ventas (POS)</string>
   </property>
  </action>
  <action name="actionGesti_n_de_clientes">
   <property name="text">
    <string>Gestión de clientes</string>
   </property>
  </action>
  <action name="actionGesti_n_de_cuentas">
   <property name="text">
    <string>Gestión de cuentas</string>
   </property>
  </action>
  <action name="actionRegistrar_pago">
   <property name="text">
    <string>Registrar pago</string>
   </property>
  </action>
  <action name="actionHistorial_pagos">
   <property name="text">
    <string>Historial pagos</string>
   </property>
  </action>
  <action name="actionInventario_actual">
   <property name="text">
    <string>Inventario actual</string>
   </property>
  </action>
  <action name="f">
   <property name="text">
    <string>Historial de ventas</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
